docs/shins/index.html

```diff
@@ -9503,23 +9503,22 @@ <h3 id="properties-41">Properties</h3>
 </tr>
 </tbody>
 </table>
-<h2 id="tocs_errorfromazure">ErrorFromAzure</h2>
-<p><a id="schemaerrorfromazure"></a>
-<a id="schema_ErrorFromAzure"></a>
-<a id="tocSerrorfromazure"></a>
-<a id="tocserrorfromazure"></a></p>
+<h2 id="tocs_errorobject">ErrorObject</h2>
```

Comment:
The `ErrorFromAzure` was a nice improvement, so I'd like to keep that. We don't need the vague `ErrorObject`.